packages/manager/modules/vps/src/dashboard/migrate/migrate.controller.js

```diff
@@ -54,4 +54,17 @@ export default class {
     return newPrices.find((price) => price.newPlan === planCode).price?.prices
       ?.originalWithoutTax.text;
   };
+
+  getPlanLabel = (planDetails) => {
+    return (
+      planDetails.planCode
+        .split('-')[1]
+        .charAt(0)
+        .toUpperCase() +
+      planDetails.planCode
+        .split('-')[1]
+        .substr(1)
```

Comment:
**nitpick:** `substr` is a deprecated method. Use `substring` instead.

Reply:
Sure Anoop